src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/PrettyFormatResponseIT.java

```diff
@@ -60,6 +64,8 @@
     private static final Set<String> messageFields = Sets.newHashSet(
             "message.dayOfWeek", "message.info", "message.author");
 
+    private static final Set<String> commentFields = Sets.newHashSet("comment.data", "comment.likes");
```

Comment:
Set.of will create unmodificable set (same for rows above and below).

https://docs.oracle.com/en/java/javase/11/core/creating-immutable-lists-sets-and-maps.html

Reply:
> Set.of will create unmodificable set (same for rows above and below).
> 
> https://docs.oracle.com/en/java/javase/11/core/creating-immutable-lists-sets-and-maps.html

Yeah, I'd love to use those new features too. Unfortunately I recall we use JDK 12 for ES build tools but still need to be compatible with JDK 8 in our code: https://github.com/opendistro-for-elasticsearch/sql/blob/master/build.gradle#L78